Localization/zh-Hans.hjson

```diff
@@ -27,7 +27,7 @@ Mods: {
 
 			Shinginzan: {
 				DisplayName: Shinginzan
-				Tooltip: "'银灰，你的盟友，前来助力。'"
+				Tooltip: "'银灰，你的盟友，前来助力。'"(未完成)
```

Comment:
Double quotation(") marks indicate the range to be displayed, while single quotation(') marks indicate lines, such as in a conversation.

So plz change like this
```suggestion
				Tooltip: "(未完成)'银灰，你的盟友，前来助力。'"
```

or you can use any colors and change profession, so

```suggestion
				Tooltip: '''
				               [c/FF0000:(未完成)]
				               ' 银灰，你的盟友，前来助力。'
				               '''
```
FF0000 is color code (red)